charts/opentelemetry-kube-stack/values.yaml

```diff
@@ -9,6 +9,18 @@ clusterName: ""
 # Extra environment variables to add to each collector, bridge and instrumentation
 extraEnvs: []
 
+# Top level field related to the OpenTelemetry Operator
+opentelemetry-operator:
+  # Field indicating whether the operator is enabled or not
+  # This is disabled for now while the chart is under development
+  enabled: false
```

Comment:
Why not enable this by default ?

Reply:
i'm waiting to enable anything until #1081 is complete as that's when this chart will actually start doing stuff